src/main/generic/utils/crypto/CryptoWorkerImpl.js

```diff
@@ -96,11 +96,53 @@ class CryptoWorkerImpl extends IWorker.Stub(CryptoWorker) {
      * @param {Uint8Array} key
      * @param {Uint8Array} salt
      * @param {number} iterations
+     * @param {number} outputSize
      * @returns {Uint8Array}
+     * @deprecated
      */
-    kdf(key, salt, iterations) {
+    kdfLegacy(key, salt, iterations, outputSize = Hash.getSize(Hash.Algorithm.ARGON2D)) {
```

Comment:
Now that you have `kdfLegacy` and `kdf`, the legacy version does not need to have the `outputSize` variable anymore, does it?